osu.Game/Screens/BackgroundScreen.cs

```diff
@@ -13,6 +13,8 @@ public abstract class BackgroundScreen : Screen, IEquatable<BackgroundScreen>
     {
         private readonly bool animateOnEnter;
 
+        public override bool IsPresent => base.IsPresent || Scheduler.HasPendingTasks;
```

Comment:
What's the reasoning for us not doing this in `Drawable` itself again?

Reply:
99% sure it was because in *some* instances we relied on drawables that are not present *not* executing their scheduled tasks (i think `NotificationManger` was the example at that time?)

Reply:
![image](https://user-images.githubusercontent.com/1329837/110277062-e5075c00-8017-11eb-9323-5820fbf3d107.png)

+ The one added in this PR... And includes a `NotificationOverlay` override >.>. I guess we'll have to see how it turns out in the future.

Reply:
Yeah, definitely something to reconsider in the future. Also see recent regression https://github.com/ppy/osu/pull/11952.